docs/source/author/topics.rst

```diff
@@ -327,6 +386,10 @@ Only curriculum areas without any children can be added to items.
 Adding a Programming Challenge
 ------------------------------------------------------------------------------
 
+There should only be one ``.yaml`` file per language directory.
+If one already exists in the language directory then add new ... to this,
```

Comment:
Maybe just put "add to this" instead of "add new … to this"? Was it a placeholder while you were thinking of a word to put there? haha. Same in other places.

Reply:
Lol copy and pasted that placeholder a lot, oops :sweat_smile: